app/controllers/orga/conferences_controller.rb

```diff
@@ -35,9 +38,9 @@ def conference
 
   def conference_params
     params[:conference] ? params.require(:conference).permit(
```

Comment:
Also ancient code. I am not sure when I should and shouldn't touch existing code that is not explicitly part of the issue I am working on. 
I really _like_ the concept of leaving code cleaner than you found it, but isn't that the opposite of doing one thing at the time? (Refactor hat or Feature hat, IIRC) . 
How do I know when to clean up, or when to leave it alone?